packages/create-app/src/createApp.ts

```diff
@@ -119,14 +119,23 @@ export default async (cmd: Command): Promise<void> => {
 
   const templateDir = paths.resolveOwn('templates/default-app');
   const tempDir = resolvePath(os.tmpdir(), answers.name);
-  const appDir = resolvePath(paths.targetDir, answers.name);
+
+  // Use `--path` argument as applicaiton directory when specified, otherwise
+  // create a directory using `answers.name`
+  const appDir = cmd.path
+    ? resolvePath(cmd.path)
+    : resolvePath(paths.targetDir, answers.name);
 
   Task.log();
   Task.log('Creating the app...');
 
   try {
-    Task.section('Checking if the directory is available');
-    await checkExists(paths.targetDir, answers.name);
+    // Directory must not already exist when using consructed path from
+    // `answers.name`
+    if (!cmd.path) {
```

Comment:
We still want to check that the directory is available when you pass in a `path` right? Or at least create the directory if it doesn't exist?

Reply:
Sure, we can verify that path already exists! Presently, if a path is specified that doesn't exist, it will be created (including subfolders).